app/views/items/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_for [@user, @user.items.build], remote: true do |f| %>
+  <div class="col-md-4">
+    <div class="form-group">
+      <%= f.label :Item %>
```

Comment:
symbols should never start with any capitalization.  Additionally, the label should match the input.